src/main/java/com/atlassian/plugins/confluence/markdown/MarkdownFromURLMacro.java

```diff
@@ -165,9 +165,13 @@ public privateRepositoryException(String message) {
 					Elements th = body.getElementsByTag("th");
 					Elements td = body.getElementsByTag("td");
 					for (Element thi : th) {
+						String alignment = thi.attributes().get("align");
```

Comment:
https://stackoverflow.com/questions/8970008/can-we-rely-on-string-isempty-for-checking-null-condition-on-a-string-in-java

Reply:
These jsoup methods as theyre written now will never return null. I can add the check in the if statement anyway just in case.


https://github.com/jhy/jsoup/blob/master/src/main/java/org/jsoup/nodes/Element.java
    @Override
    public Attributes attributes() {
        if (!hasAttributes())
            attributes = new Attributes();
        return attributes;
    }

https://github.com/jhy/jsoup/blob/master/src/main/java/org/jsoup/nodes/Attributes.java
    /**
     Get an attribute value by key.
     @param key the (case-sensitive) attribute key
     @return the attribute value if set; or empty string if not set (or a boolean attribute).
     @see #hasKey(String)
     */
    public String get(String key) {
        int i = indexOfKey(key);
        return i == NotFound ? EmptyString : checkNotNull(vals[i]);
    }

    static String checkNotNull(String val) {
        return val == null ? EmptyString : val;
    }